src/app/book/add/add.component.ts

```diff
@@ -0,0 +1,61 @@
+import {Component, OnInit} from '@angular/core';
+import {Book} from '../../models/book.model';
+import {BookService} from '../../services/book.service';
+import {Router} from '@angular/router';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+
+@Component({
+  selector: 'app-add',
```

Comment:
You can rename the selector `book-add`  in order to make it more human-friendly.